### PR TITLE
Guest role gherkin .feature files.

### PR DESCRIPTION
**Pull request by mrpau-richard**
### Summary
I create a gherkin .feature files that contains a guest role.

### Contributor Checklist

- [x] Contributor has fully tested the PR manually
- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

File: integration_testing/features/guest/guest-engage-content-channels.feature [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Feature: Guest engage content channels
+  Guest engage with content on channels page
+
+  Background:
+    Given I am on *user > sign in* page
+      And I click *Continue as guest* button
+      And There is a channel <channel> that contains exercises
+      And I am on *learn > classes page
```
Comment (radinamatic) [inline integration_testing/features/guest/guest-engage-content-channels.feature]: If this is a guest user, they cannot be on *Learn > Classes* page.

Comment (radinamatic) [reply integration_testing/features/guest/guest-engage-content-channels.feature]: Remember always to capitalize the location, so `*Learn > Channels*` or `*Learn > Recommended*`.

File: integration_testing/features/guest/guest-engage-content-channels.feature [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  Scenario: Guest engage with content on channels page
+    When I select <channel>
+     And I am on *learn > topic* page
```
Comment (radinamatic) [inline integration_testing/features/guest/guest-engage-content-channels.feature]: The convention we are using is meant to indicate the location with plugin and tab specifiers, and not go further down the channel's topic tree, so this is technically still `*Learn > Channels*`.

File: integration_testing/features/guest/guest-engage-content-recommended.feature [hunk 3]
```diff
@@ -0,0 +1,19 @@
+Feature: Guest engage content recommended
+  Guest engage with content on recommended page
+
+  Background:
+    Given I am on *user > sign in* page
+      And I click *Continue as guest* button
+      And There is a channel <channel> that contains videos and exercises
+      And I am on *learn > classes page
```
Comment (radinamatic) [inline integration_testing/features/guest/guest-engage-content-recommended.feature]: Same comments as above: remember to capitalize, and no `Classes` page for guests.

File: integration_testing/features/guest/guest-find-content-item.feature [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  Guest find a specific content item using the search bar
+
+  Background:
+    Given I am on *user > sign in* page
+      And I click *Continue as guest* button
+      And There is a channel <channel> that contains videos and exercises
+      And I am on *learn > classes page
+
+  Scenario: Guest find a specific content item using the search bar
+    When I fill in the search bar <search bar> with the item that I need to find
```
Comment (radinamatic) [inline integration_testing/features/guest/guest-find-content-item.feature]: This would be better named as `search_term`, as *search bar* is an UI element.

**Review by radinamatic — CHANGES_REQUESTED** [review]
Added a couple of comments, but otherwise looking good!

**Comment by mrpau-richard** [comment]
@radinamatic done fixing all of your review comments in this PR.

**Review by radinamatic — APPROVED** [review]

**Pull request merged**